types/git-diff/index.d.ts

```diff
@@ -19,8 +19,11 @@ declare namespace gitDiff {
          */
         color?: boolean;
         /**
-         * A space separated string of git diff flags from https://git-scm.com/docs/git-diff#_options
+         * A space separated string of git diff flags.
+         * 
```

Comment:
"Trailing whitespace" linting issue here

Reply:
Alright, just removed the space right next to asterisk char